pkg/pipeline/transform/kubernetes/enrich.go

```diff
@@ -17,12 +17,17 @@ func MockInformers() {
 	informers = inf.NewInformersMock()
 }
 
-func InitFromConfig(kubeConfigPath string) error {
-	return informers.InitFromConfig(kubeConfigPath)
+func InitFromConfig(config api.NetworkTransformKubeConfig) error {
+	return informers.InitFromConfig(config)
 }
 
 func Enrich(outputEntry config.GenericMap, rule api.K8sRule) {
-	kubeInfo, err := informers.GetInfo(fmt.Sprintf("%s", outputEntry[rule.Input]))
+	ip := fmt.Sprintf("%s", outputEntry[rule.Input])
+	// TODO: replace this by a new rule input
+	mac := fmt.Sprintf("%s", outputEntry[strings.Replace(rule.Input, "Addr", "Mac", -1)])
```

Comment:
This need to be changed before merge
I'm still discussing about putting this behind a feature gate or not at this stage

Reply:
https://github.com/netobserv/flowlogs-pipeline/pull/674/commits/5c0f987bffdf1c59257291c36635e389809c7ad2